app/src/main/java/com/example/joshuarodriguez/mvpcalculator/mvp/mvp/view/CalculatorView.java

```diff
@@ -56,136 +56,134 @@
     @BindView(R.id.button_equals) Button mButtonEquals;
     @BindView(R.id.button_clear) Button mButtonClear;
 
-    public CalculatorView(AppCompatActivity activity, Bus eventBus) {
+    public CalculatorView(AppCompatActivity activity) {
         super(activity);
-        this.eventBus = eventBus;
         ButterKnife.bind(this, activity);
-        hideKeyboard();
     }
 
     @OnClick(R.id.button_number_zero)
     public void numericButtonZeroPressed() {
-        eventBus.post(new CalculatorNumericButtonPressedEvent(
-                ViewUtil.getStringResourceValue(getActivity(), R.string.calculator_button_value_zero)
+        RxBus.post(new CalculatorNumericButtonPressedObserver.NumericButtonValue(
+                getStringResource(R.string.calculator_button_value_zero)
         ));
```

Comment:
Okay understood. The only problem I see here is that the presenter has no way of really determining what value to use if I am just posting: 

new CalculatorNumericButtonPressedObserver.NumericButtonValue()

One way I am currently thinking of is creating several Observer classes that correspond to each button and then getting the string resource in the presenter. However, I feel like creating inner classes for each button is not the way to go as it would be quite verbose. Otherwise, this would work.

Any suggestions?